Makefile.am

```diff
@@ -2,6 +2,10 @@ AUTOMAKE_OPTIONS=foreign
 
 AM_CFLAGS=-pedantic -Wall -Werror -Wextra
 
+if LINUX
```

Comment:
I see. Thanks for the links! I'm happy to try this and correct course if we run into issues with some distro.